src/index/index.h

```diff
@@ -48,6 +49,7 @@ typedef struct {
 	double weight;     // the importance of text
 	bool nostem;       // disable stemming of the text
 	char *phonetic;    // phonetic search of text
+	int ref_count;     // reference count of the field
```

Comment:
Can the `IndexField` have a `ref_count` value of `-150`, and would it make sense? What's the benefit of allowing the range of negative values and requiring more value checks for this value?

Reply:
See this line:
https://github.com/RedisGraph/RedisGraph/blob/78fa837b8b5e1b26423eb469254e790bd6913a92/src/index/index.c#L349